packages/@uppy/companion/src/server/middlewares.js

```diff
@@ -1,5 +1,6 @@
 const tokenService = require('./helpers/jwt')
 const logger = require('./logger')
+const uniq = require('lodash/uniq')
```

Comment:
`lodash` is not a dependency of Companion yet, it just happens to be available in the repository due to hoisted dependencies. it should be added to its package.json.

Reply:
Whoops! Nice catch. I misread this line as lodash/merge assuming that lodash was already there:

https://github.com/transloadit/uppy/blob/2d73620b5b3d0e7bfe50c9a0f257f567af9338ac/packages/%40uppy/companion/src/companion.js#L13

Reply:
i guess we could also use `lodash.uniq` here or change both to `lodash/{uniq,merge}`, whatever works :)

Reply:
Also `import/no-extraneous-dependencies` would have caught this error. Have an ongoing issue about improving eslint rules here: https://github.com/transloadit/node-sdk/issues/90

Reply:
Import lints would actually be really useful for Uppy. We've had a few cases where we cut a new release with missing dependencies. (I thought `eslint-plugin-import` only worked with import syntax for some reason, but it looks like it will also work for `require()`!)

Reply:
Yes, it's such an easy mistake to make. that rule saved my ass so many times, and this is my [2nd time](https://github.com/transloadit/node-sdk/issues/93) making the same mistake here in a few weeks now

Reply:
If you have the time and interest, it would be great if you could send a PR that adds that :D 

I have looked into using that dependency-check program that Julian mentioned, but it was a bit awkward with a big monorepo. Hopefully the import lint plugin _can_ deal with that.